etc/alembic.ini

```diff
@@ -0,0 +1,116 @@
+# A generic, single database configuration.
```

Comment:
It felt like adding the alembic.ini to `etc/` seemed like the right place. But this means when we call alembic we have to specify the config's location.

```
/opt/repos/uta# alembic --config etc/alembic.ini upgrade head
```